src/components/Launch/LaunchWorkflowForm/inputConverters.ts

```diff
@@ -0,0 +1,62 @@
+import { dateToTimestamp, millisecondsToDuration } from 'common/utils';
+import { Core } from 'flyteidl';
+import * as Long from 'long';
+import * as moment from 'moment';
```

Comment:
It doesn't look like this import is used. Also, I think you've capitalized `Moment` elsewhere when importing it in this fashion.

Reply:
It's used on line 28. My default behavior is to keep the import following the TSLint naming conventions (camelCased import). The one other place I do this, I'm importing and assigning `moment.utc` as `moment` so that all usages in the rest of the file are guaranteed to use the utc versions. I probably should do that here as well!